contributing/contributing-code.md

```diff
@@ -123,18 +123,12 @@ For further detail on integrating third-party services (e.g., fonts, embeds, etc
 
 ## Contributor License Agreement
 
-AMP requires all contributors to either sign an individual Contributor License Agreement or be covered by a corporate Contributor License Agreement in order to protect contributors and users in issues of intellectual property.
+AMP requires all contributors to agree to the OpenJSF Contributor License Agreement in order to protect contributors and users in issues of intellectual property.
 
-We recommend you handle signing/being covered by a CLA _before_ you send a PR to avoid problems, though this is not absolutely necessary until your code is ready to be merged in.
+To agree to the OpenJSF Contributor License Agreement, visit https://cla-assistant.io/ampproject/amphtml, read through the agreement, and click "Login with GitHub to Agree."
 
 **Make sure that the email you associate with your CLA is the same email address you associate with your commits (likely via the `user.email` Git config as described on GitHub's [Set up Git](https://help.github.com/articles/set-up-git/) page).**
```

Comment:
@rsimha is this still the case?  Or does CLA assistant work based on the GitHub id?

@tobie in case he knows as well

Reply:
It uses your Github id, so you can get rid of this altogether.

Reply:
Can confirm that the check is based on GitHub ID.

Reply:
I took the liberty of adding a commit to remove this sentence.